schemas/operators/accessor/missingSome.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
http://jsonlogic.com/operations.html#missing_some

Reply:
@lina-salih 
@mjenny 
May we should strictly keep the original underscore-case 🤔 
`missingSome` to `missing_some`, same for all other files

Reply:
it should be consistent :)

Reply:
I renamed this one, all others within a separate PR